packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-form.jsx

```diff
@@ -22,109 +22,150 @@ const ProofOfOwnershipForm = ({
+                        (!is_pm_identifier_provided &&
+                            total_documents_uploaded === payment_method?.documents_required * 0.5)
+                    ) {
+                        are_files_uploaded = false;
                     }
-                    if (file?.size / 1024 > 8000) {
-                        errors.data[card_key][item_key].files[i] = localize(
-                            'That file is too big (only up to 8MB allowed). Please upload another file.'
-                        );
-                    }
-                    if (!is_pm_identifier_provided) {
+                    delete errors.data[card_key][item_key].payment_method_identifier;
+                    payment_method?.files?.forEach((file, i) => {
+                        errors.data[card_key][item_key].files = errors?.data?.[card_key]?.[item_key]?.files ?? [];
+                        if (file?.type && !/(image|application)\/(jpe?g|pdf|png)$/.test(file?.type)) {
+                            errors.data[card_key][item_key].files[i] = localize(
+                                "That file format isn't supported. Please upload .pdf, .png, .jpg, or .jpeg files only."
+                            );
+                        }
+                        if (file?.size / 1024 > 8000) {
+                            errors.data[card_key][item_key].files[i] = localize(
+                                'That file is too big (only up to 8MB allowed). Please upload another file.'
+                            );
+                        }
+                        if (errors.data[card_key][item_key].files.length === 0) {
+                            delete errors.data[card_key][item_key].files;
+                        }
+                        if (
+                            !is_pm_identifier_provided &&
+                            (total_documents_uploaded === payment_method?.documents_required ||
+                                (!payment_method?.documents_required && total_documents_uploaded > 0) ||
+                                total_documents_uploaded === payment_method?.documents_required * 0.5)
+                        ) {
+                            errors.data[card_key][item_key].payment_method_identifier =
+                                localize('Please complete this field.');
+                        }
+                    });
+                    if (
+                        (is_credit_or_debit_card &&
+                            payment_method_identifier?.length !== 0 &&
+                            (payment_method_identifier?.length !== 16 || payment_method_identifier?.length > 19) &&
+                            !VALIDATIONS.is_formated_card_number(payment_method_identifier)) ||
+                        (is_credit_or_debit_card &&
+                            payment_method_identifier?.length === 16 &&
+                            VALIDATIONS.has_invalid_characters(payment_method_identifier))
+                    ) {
```

Comment:
```suggestion
                    if (
                        is_credit_or_debit_card &&
                        ((payment_method_identifier?.length !== 0 &&
                            (payment_method_identifier?.length !== 16 || payment_method_identifier?.length > 19) &&
                            !VALIDATIONS.is_formated_card_number(payment_method_identifier)) ||
                            (payment_method_identifier?.length === 16 && VALIDATIONS.has_invalid_characters(payment_method_identifier)))
                    ) {
```
@jim-deriv just another nit-pick in order to not put is_credit_or_debit_card twice😉